src/js/components/ServiceForm.js

```diff
@@ -104,6 +104,40 @@ class ServiceForm extends SchemaForm {
     this.props.onChange(...arguments);
   }
 
+  validateForm() {
+    let model = this.triggerTabFormSubmit();
+
+    // Clears all previous errors.
+    FormUtil.forEachDefinition(this.multipleDefinition, function (definition) {
```

Comment:
@mlunoe I can't agree more with you! We should never store any data on components itself.  Let's get rid of this one during the schema form "rewrite", which is scheduled for the next release if I remember it correctly...